README.md

```diff
@@ -351,7 +351,7 @@ session data has been altered (though this behavior can be altered with various
 options in the middleware constructor). Because of this, typically this method
 does not need to be called.
 
-There are some cases where it is useful to call this method, for example, long-
+There are some cases where it is useful to call this method, for example, redirects, long-
```

Comment:
Love it! Just need to fix the wrapping to be back to 80 chars. Just wrap the redirect down to the next line :)